### PR TITLE
added registry console section per BZ

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1518204

### CODE REVIEW

**Comment by mburke5678** [comment]
@tripledes I am not clear on the `openshift_cockpit_deployer_prefix` setting. Can the prefix be any test string or does it point to an existing image and path?

**Comment by mburke5678** [comment]
@tripledes PTAL

**Comment by tripledes** [comment]
@mburke5678 LGTM, @jtudelag works?

**Review by jtudelag — COMMENTED** [review]
@mburke5678 @tripledes Please review ;)

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -784,6 +783,38 @@ openshift_examples_modify_imagestreams=true
...
+|`*openshift_cockpit_deployer_version*`
+|Specify the Cockpit image verion
+
+|`*openshift_cockpit_deployer_prefix*`
+|Specify the image prefix, a string of text added to the beginning of the image name, to indicate the image location.
+|===
+
+For example:
+----
+openshift_cockpit_deployer_prefix='registry.example.com/openshift'
```
[the remark this answers is not in the document]
Comment (tripledes) [reply install_config/install/advanced_install.adoc]: Based on https://github.com/openshift/openshift-ansible/blob/release-3.6/inventory/byo/hosts.ose.example#L416-L423 I'd say what @mburke5678 is correct. @mburke5678 could you please ping anyone from openshift-ansible to confirm?

[the remark this answers is not in the document]
Comment (mburke5678) [reply install_config/install/advanced_install.adoc]: @sdodson Can you help? We are trying to determine an appropriate example for the `openshift_cockpit_deployer_prefix` parameter. 
`openshift_cockpit_deployer_prefix='registry.example.com/openshift' `
`openshift_cockpit_deployer_prefix='registry.example.com/openshift/console'`
Other?

**Comment by mburke5678** [comment]
@jianlinliu PTAL. Can you help answer the question above that I asked @sdodson about?

File: install_config/install/advanced_install.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (sdodson) [reply install_config/install/advanced_install.adoc]: If your image is at `registry.example.com/openshift3/registry-console` then it should be `openshift_cockpit_deployer_prefix='registry.example.com/openshift3/'` as `registry-console` is appended on to the end, though that too can be modified via `openshift_cockpit_deployer_basename`

File: install_config/install/advanced_install.adoc [hunk 2]
```diff
@@ -784,6 +783,44 @@ openshift_examples_modify_imagestreams=true
...
+[[advanced-install-configuring-registry-location]]
+=== Configuring the Registry Console
+
+If you are using a registry console image other than the default or require a specific version of the console,
+specify the desired registry within the
+*_/etc/ansible/hosts_* file. 
+
+----
+openshift_cockpit_deployer_prefix=<registry-name>/<namespace>/
+openshift_cockpit_deployer_basename=<my-console>
```
Comment (jianlinliu) [inline install_config/install/advanced_install.adoc]: This PR is targeted for 3.7, right? If yes, then it is okay. "openshift_cockpit_deployer_basename" is not supported yet in 3.6 openshift-ansible code yet.

File: install_config/install/advanced_install.adoc [hunk 3]
```diff
@@ -784,6 +783,44 @@ openshift_examples_modify_imagestreams=true
...
+|`*openshift_cockpit_deployer_version*`
+|Specify the Cockpit image verion
+|===
+
+For example:
+If your image is at *_registry.example.com/openshift3/registry-console_* and you require
+version 1.4.1, enter:
+----
+openshift_cockpit_deployer_prefix='registry.example.com/openshift3/' 
+openshift_cockpit_deployer_basename=`registry-console`
```
Comment (jianlinliu) [inline install_config/install/advanced_install.adoc]: "registry-console" is the default value, in this example, it is better to change it to other than the default name

**Comment by mburke5678** [comment]
@tripledes Can you comment on @jianlinliu comment about which version that `openshift_cockpit_deployer_basename` was added? ^^ 
I removed `openshift_cockpit_deployer_basename` from this PR and do a follow-up to add the parameter to 3.7+, if needed.

**Comment by mburke5678** [comment]
[rev_history]
|xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Advanced Installation]
|Added information on xref:../install_config/install/advanced_install.adoc#configuring-a-registry-location[using a Cockpit registry console image other than the default].
%

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL as we settle on versioning.

**Comment by jianlinliu** [comment]
LGTM.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

**Comment by ahardin-rh** [comment]
@mburke LGTM. In the revision history, you just need to update the first line to show:
[Installing a Cluster -> Advanced Installation] 🎆

File: install_config/install/advanced_install.adoc [hunk 4]
```diff
@@ -784,6 +783,38 @@ openshift_examples_modify_imagestreams=true
 openshift_docker_additional_registries=example.com
 ----
 
+[[advanced-install-configuring-registry-location]]
```
Comment (vikram-redhat) [inline install_config/install/advanced_install.adoc]: @mburke5678 this is a duplicate id, which seems to be copied from the previous section. I have changed it to advanced-install-configuring-registry-console.

**Comment by vikram-redhat** [comment]
@mburke5678 - the rev history link to the section id was not correct. Instead of configuring-a-registry-location it should have been advanced-install-configuring-registry-console.

**Pull request merged**